### PR TITLE
Lu 44 research

### PR DESCRIPTION
**Pull request by Aman9804**
Please Review and Approve the changes for MDXParser research

### CODE REVIEW

File: src/components/MDXParse/mdxp.tsx [hunk 1]
```diff
@@ -1,37 +1,34 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
 "use client";
 import React, { forwardRef } from "react";
 import { cx } from "@utils/cx";
 import StringToReactComponent from 'string-to-react-component';
```
Comment (archan96) [inline src/components/MDXParse/mdxp.tsx]: Can this be forked and convert to internal util?

**Review by archan96 — APPROVED** [review]
Seems alright. Just let me know about the fork possibilities of  "string-to-react-component"

File: src/components/MDXParse/mdxp.tsx (hunk 1 above)
Comment (Aman9804) [reply src/components/MDXParse/mdxp.tsx]: Sure, I will add it to internal unit as an util and commit again.

**Comment by Aman9804** [comment]
@archan96 string-to-react-component library removed and used as an util. Please check and approve

**Review by xynes-admin — APPROVED** [review]
Seems Alright

**Pull request merged**